docs/developer/contributing/index.asciidoc

```diff
@@ -0,0 +1,92 @@
+
+[float]
+[[kibana-localization]]
+=== Localization
+
+Read <<kibana-localization-best-practices>> for details on our localization practices.
+
+Note that we cannot support accepting contributions to the translations from any source other than the translators we have engaged to do the work.
+We are still to develop a proper process to accept any contributed translations. We certainly appreciate that people care enough about the localization effort to want to help improve the quality. We aim to build out a more comprehensive localization process for the future and will notify you once contributions can be supported, but for the time being, we are not able to incorporate suggestions.
+
+[float]
+[[kibana-release-notes-process]]
+=== Release Notes Process
+
+Part of this process only applies to maintainers, since it requires
+access to GitHub labels.
+
+{ kib} publishes https://www.elastic.co/guide/en/kibana/current/release-notes.html[Release Notes] for major and minor releases. 
+The Release Notes summarize what the PRs accomplish in language that is meaningful to users.
+ To generate the Release Notes, the team runs a script against this repo to collect the merged PRs against the release.
+
+[float]
+==== Create the Release Notes text
+
+The text that appears in the Release Notes is pulled directly from your PR title, or a single paragraph of text that you specify in the PR description.
+
+To use a single paragraph of text, enter `Release note:` or a `## Release note` header in the PR description, followed by your text. For example, refer to this [PR](https://github.com/elastic/kibana/pull/65796) that uses the `## Release note` header.
+
+When you create the Release Notes text, use the following best practices:
+* Use present tense. 
+* Use sentence case.
+* When you create a feature PR, start with `Adds`.
+* When you create an enhancement PR, start with `Improves`. 
+* When you create a bug fix PR, start with `Fixes`.
+* When you create a deprecation PR, start with `Deprecates`.
+
+[float]
+==== Add your labels
+
+[arabic]
+. Label the PR with the targeted version (ex: `v7.3.0`).
+. Label the PR with the appropriate GitHub labels:
+  * For a new feature or functionality, use `release_note:enhancement`.
+  * For an external-facing fix, use `release_note:fix`. We do not include docs, build, and test fixes in the Release Notes, or unreleased issues that are only on `master`.
+  * For a deprecated feature, use `release_note:deprecation`.
+  * For a breaking change, use `release_note:breaking`.
+  * To **NOT** include your changes in the Release Notes, use `release_note:skip`.
+
+....
+# Dev Docs
```

Comment:
Should this be removed?  It looks out of place in the rendered doc.

Reply:
@KOTungseth - what do you think about this section? I think you recently added it to the CONTRIBUTING guide and I moved here.  Maybe give a title "For example, include these sections in your git issue:"?

Reply:
I agree with Peter, we probably don't need it here. I think this is more for breaking changes.